README.md

```diff
@@ -5,13 +5,13 @@ This uses [`mkdocs`](https://www.mkdocs.org/)
 Install it with:
 
 ```
-pip install mkdocs
+pip3 install mkdocs-material
```

Comment:
the README.md was updated on the master branch already. Also, this is using pip3 for python 2, which is incorrect.